ykrt/src/compile/jitc_yk/jit_ir.rs

```diff
@@ -238,16 +232,15 @@ impl Module {
         &self.insts
     }
 
-    /// Push a slice of extra arguments into the extra arg table.
+    /// Push a slice of arguments into the args pool.
     ///
     /// # Panics
     ///
-    /// If `ops` would overflow the index type.
-    fn push_extra_args(&mut self, ops: &[Operand]) -> Result<ExtraArgsIdx, CompilationError> {
-        assert!(ExtraArgsIdx::new(self.types.len()).is_ok());
-        let idx = self.extra_args.len();
-        self.extra_args.extend_from_slice(ops); // FIXME: this clones.
-        ExtraArgsIdx::new(idx)
+    /// If `args` would overflow the index type.
+    fn push_args(&mut self, args: &[Operand]) -> Result<ArgsIdx, CompilationError> {
+        let idx = self.args.len();
+        self.args.extend_from_slice(args); // FIXME: this clones.
```

Comment:
I was staring at this, scratching my head thinking, "I don't see a `self.args`"... I must be losing the plot.

Checked out your branch, and indeed, `self.args` is only introduced in the next commit!

Reply:
Erp. I might have made a mistake when breaking this up. I can probably fix this in a rebase.

Reply:
If it isn't too annoying, please go ahead. Might make it easier to review.

Reply:
OK, force pushed an update. Once I realised what (silly) mistake I'd made in splitting things up, it wasn't too bad!

Reply:
Thanks!